### PR TITLE
IBX-22: Added danger font color

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-22
| Related to | https://github.com/ezsystems/ezplatform-page-builder/pull/740
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_custom.scss [hunk 1]
```diff
@@ -28,6 +28,7 @@ $ibexa-color-font: #131c26;
 $ibexa-color-font-light: #fff;
 $ibexa-color-font-pale: #878b90;
 $ibexa-color-font-black: #222222;
+$ibexa-color-font-danger:#d92d42;
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_custom.scss:31]: I am really sorry, but... :D 
```suggestion
$ibexa-color-font-danger: #d92d42;
```

**Review by tischsoic — CHANGES_REQUESTED** [review]

**Review by tischsoic — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA approved on Ibexa Commerce 3.3.2-dev with diff.

**Comment by adamwojs** [comment]
Could you please merge up changes @lucasOsti?

**Comment by ViniTou** [comment]
I had merged it up as it dependency for page-builder & page-fieldtype PRs.

**Pull request merged**